docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md

```diff
@@ -95,6 +96,37 @@ To create an export configuration, do the following:
 1. Enter your installation token, access ID, and access key.
 1. Optionally, click **Download** to download the Sumo Logic dashboard template. After you install the [YugabyteDB app](https://help.sumologic.com/docs/get-started/apps-integrations/) (coming soon) in your Sumo Logic account, you can import this JSON format template and use it as a starting point for visualizing your cluster data.
 1. Click **Test Configuration** to make sure your connection is working.
+1. Click **Create Configuration**.
+
+  {{% /tab %}}
+
+  {{% tab header="Prometheus" lang="prometheus" %}}
+
+Prometheus integration is {{<badge/tp>}} and only available for clusters deployed on AWS.
+
+The Prometheus integration requires the following:
+
+- Prometheus instance
+  - deployed in a VPC on AWS
+  - [OLTP Receiver](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver) feature flag enabled
+  - publically-accessible endpoint URL; the DNS for the endpoint must be in a public hosted zone in AWS.
+- YugabyteDB cluster from which you want to export metrics
+  - the cluster is [deployed in VPCs](../../cloud-basics/cloud-vpcs/cloud-add-vpc/) on AWS
+  - each region VPC is peered with the VPC hosting Prometheus. See [Peer VPCs](../../cloud-basics/cloud-vpcs/cloud-add-vpc-aws/).
+  - VPCs have the following Inbound Security Group rules:
+    - Allow HTTP inbound traffic on port 80 for Prometheus endpoint URL (HTTP)
+    - Allow HTTPS inbound traffic on port 443 for Prometheus endpoint URL (HTTPS)
+
+    See [Control traffic to your AWS resources using security groups](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-security-groups.html) in the AWS documentation.
+
+Note that VPC requirements apply to all regions in multi-region clusters in AWS. See [VPC network overview](../../cloud-basics/cloud-vpcs/cloud-vpc-intro/).
```

Comment:
In general in our documentation when we mention multi-region clusters does it include GEO, Read Replica clusters as well? If not should we specify those as well here?